elastic/kibana
# [ILM] Surfacing policy error state

## Summary

Fix https://github.com/elastic/kibana/issues/87520

Adds top-level indications of ILM policy form error states for the collapsible phases.

### How to test

1. Start kibana on basic
2. Navigate to ILM policy plugin in the management section
3. Edit or create a new policy
4. Activate each phase, introducing some validation error in each. Usually a negative number in force merge or replicas will do.
5. Collapse all the phases and make sure the red alert icon is showing on all phases with validation errors and the global error callout at the bottom of the form is showing while there are any validation errors.

### Screenshots

<img width="1120" alt="Screenshot 2021-01-28 at 14 52 07" src="https://user-images.githubusercontent.com/8155004/106267502-095a6680-622a-11eb-997d-38fe6a3a77e5.png">

<img width="1156" alt="Screenshot 2021-01-28 at 14 52 18" src="https://user-images.githubusercontent.com/8155004/106267509-0bbcc080-622a-11eb-9d69-f2354365f10e.png">


### Checklist

Delete any items that are not applicable to this PR.
